ElectronClient/InteropServiceHelper.js

```diff
@@ -107,9 +108,14 @@ class InteropServiceHelper {
 		return this.exportNoteTo_('printer', noteId, options);
 	}
 
-	static async defaultFilename(noteId, fileExtension) {
-		if (!noteId) return '';
-		const note = await Note.load(noteId);
+	static async defaultFilename(noteIds, fileExtension) {
```

Comment:
Function signature shouldn't change (it accepts a note ID, not an array)

Reply:
Thanks for the catch, I guess thats something that has changed since I first made the pull request. I've fixed it now.